doc/qos_overview.md

```diff
@@ -422,3 +429,176 @@ markings. The same interface has an egress scheduler policy applied to it.
   }
 }
 ```
+
+### Ingress Classification with Ingress Scheduling (Policer)
+
+The example QoS configuration below shows the configuration of an interface,
+assumed to be facing a customer which has ingress classification based on
+the next-hop group the packet is being sent to.  The same interface has an
+ingress scheduler policy applied to it which implementes a ONE_RATE_TWO_COLOR
+policer.
+
+In this scenario, the device is does not have hardware or software to implement
```

Comment:
We should clarify that on a VOQ architecture this is needed. It's not a "dummy" queue if we have the ability to do this queueing on an egress linecard.

Reply:
Added more description for this